app/commands/donations/subscription/cancel.rb

```diff
@@ -15,10 +15,6 @@ def call
       raise if data.status == 'active'
     end
 
-    subscription.update!(active: false)
-
-    # Update based on whether there is another different active subscription
-    user = subscription.user
-    User::SetActiveDonationSubscription.(user, user.donation_subscriptions.active.exists?)
+    Donations::Subscription::Deactivate.(subscription)
```

Comment:
I've extracted this logic to a separate command as I want to use its functionality in the syncing command too